tests/dialects/test_snowflake.py

```diff
@@ -821,6 +821,13 @@ def test_snowflake(self):
                 "snowflake": "CASE WHEN x = a OR (x IS NULL AND a IS NULL) THEN b WHEN x = c OR (x IS NULL AND c IS NULL) THEN d ELSE e END",
             },
         )
+        self.validate_all(
+            "SELECT LISTAGG(col1, ', ') WITHIN GROUP (ORDER BY col2) FROM t",
+            read={"snowflake": "SELECT LISTAGG(col1, ', ') WITHIN GROUP (ORDER BY col2) FROM t"},
+            write={
+                "duckdb": "SELECT GROUP_CONCAT(col1, ', ' ORDER BY col2) FROM t",
```

Comment:
Does the duckdb -> snowflake direction work?

Reply:
Doesn't seem like it sadly. Should DuckDB parse ordered agg functions as `exp.WithinGroup` now that the generation is supported, just so it can generalize across other dialects like Snowflake, Postgres etc?

Reply:
We can leave it for another PR to avoid digressing here.